### PR TITLE
[OTA-2468] clean up the other pages

### PR DESCRIPTION
**Pull request by piotrcichonmee**
Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/containers/Updates.jsx [hunk 1]
```diff
@@ -4,19 +4,32 @@ import PropTypes from 'prop-types';
...
+
   @observable
   updateDetailsShown = false;
+
   @observable
   selectedUpdate = null;
 
+  constructor(props) {
+    super(props);
+    this.state = { pageNumber: PAGE_NUMBER_DEFAULT };
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/src/containers/Updates.jsx]: Any particular reason to have `state` declared in `constructor` in this case over shorthand equivalent?
```suggestion
    state = {
      pageNumber: PAGE_NUMBER_DEFAULT
    };
```

Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/containers/Updates.jsx]: Please see: https://reactjs.org/docs/state-and-lifecycle.html

File: ota-plus-web/app/reactapp/src/components/software/List.jsx [hunk 2]
```diff
@@ -2,15 +2,15 @@
 
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { observable, observe } from 'mobx';
+import { observable, observe, toJS } from 'mobx';
```
[the remark this answers is not in the document]
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/software/List.jsx]: Done

File: ota-plus-web/app/reactapp/src/components/updates/Header.jsx [hunk 3]
```diff
@@ -2,28 +2,63 @@
...
-        <div className='update-subheader__item'>Title</div>
-        <div className='update-subheader__item'>Internal description</div>
-        <a href='#' className='add-button grey-button' id='add-new-update' onClick={showCreateModal}>
-          <span>+</span>
-          <span>Create new update</span>
-        </a>
-      </SubHeader>
+      <div>
+        <div className="tab-navigation">
+
```
[the remark this answers is not in the document]
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/updates/Header.jsx]: Done

File: ota-plus-web/app/reactapp/src/components/software/List.jsx [hunk 4]
```diff
@@ -183,104 +192,98 @@ class List extends Component {
...
-                                        }}
-                                      >
-                                        <img src='/assets/img/icons/trash_icon.svg' alt='Icon' />
-                                        Delete
-                                      </a>
-                                    </li>
-                                  </Dropdown>
-                                ) : null}
-                              </div>
+                              {this.submenuIsShown ? (
```
[the remark this answers is not in the document]
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/software/List.jsx]: Done

File: ota-plus-web/app/reactapp/src/components/software/List.jsx [hunk 5]
```diff
@@ -183,104 +192,98 @@ class List extends Component {
...
+                        animation: 'slideUp',
+                        begin: () => {
+                          that.startIntervalListScroll();
+                        },
+                        complete: () => {
+                          that.stopIntervalListScroll();
+                        },
+                      }}
+                    >
+                      {expandedPackageName === pack.packageName ? (
```
[the remark this answers is not in the document]
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/software/List.jsx]: Done

File: ota-plus-web/app/reactapp/src/components/updates/Header.jsx [hunk 6]
```diff
@@ -2,28 +2,63 @@
...
+            </Form>
+            <Button
+              htmlType="button"
+              className="ant-btn ant-btn-hero"
+              id="add-new-update"
+              onClick={() => {
+                showCreateModal(null);
+              }}
+            >
+              {'Add software'}
```
[the remark this answers is not in the document]
Comment (piotrcichonmee) [reply ota-plus-web/app/reactapp/src/components/updates/Header.jsx]: Done

**Pull request merged**